iib/workers/tasks/opm_operations.py

```diff
@@ -415,6 +420,59 @@ def opm_generate_dockerfile(
     return dockerfile_path
 
 
+def insert_cache_into_dockerfile(dockerfile_path: str, local_cache_path: str) -> None:
+    """
+    Insert built cache into the Dockerfile.
+
+    :param str dockerfile_path: path to the generated Dockerfile.
+    :param str local_cache_path: path to the locally generated cache.
+    """
+    with open(dockerfile_path, 'r') as f:
+        file_data = f.read()
+
+    file_data = file_data.replace(
+        'RUN ["/bin/opm", "serve", "/configs", "--cache-dir=/tmp/cache", "--cache-only"]',
+        f'COPY {local_cache_path} /tmp/cache',
+    )
```

Comment:
Question: Are we just replacing the local Dockerfile but not building the final image with this Dockerfile, right?

Because I believe the final image must serve from its own cache and here we're replacing the `RUN` serve command with just a `COPY` of contents from our cache to the final image cache...

My fear is that when we build the final image it will not have the correct `RUN` line...

Reply:
good question, I don't think the final index needs to have the `RUN` line since the RUN line was actually building the cache. We are directly copying the built cache into the location OPM expects it to be. That's my understanding. 

I'll request Joe Lanford to take a look at this too just to be sure

Reply:
@JAVGan We are not replacing the run of `opm serve` - it is still triggered as CMD inside the Dockerfile.
This replaces only generation of cache files (see `--cache-only` parameter).

Reply:
Yep, @yashvardhannanavati and @lipoja are correct!

Reply:
Thanks!